site/_posts/2021-10-26-how-to-set-up-single-sign-on-between-fusionauth-wordpress.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog-post
+title: How to Set Up Single Sign-On Between FusionAuth and WordPress
+description: In this tutorial, you'll be learning how to implement OAuth SSO for Joomla users.
+author: Aniket Bhattacharyea
+image: blogs/wordpress-sso-fusionauth/how-to-set-up-single-sign-on-sso-between-fusionauth-and-wordpress-header-image.png
+category: blog
+tags: tutorial tutorial-php client-php tutorial-integration
+excerpt_separator: "<!--more-->"
+---
+
+User authentication is one of the most important parts of building any application, and almost every application features some sort of authentication. However, it quickly gets difficult to manage when you have multiple related but independent applications to maintain.
+
+<!--more-->
+
+Let's say you're building a photo-sharing application that lets a user sign up with a username and password to share their pictures. Your team might later decide to [create a forum using WordPress](https://wordpress.com/go/tutorials/create-a-forum-with-wordpress-com/) to help support your app, but this new project isn't tied to your original application. It uses an entirely different tech stack, database, and hosting setup.
+
+This means your users need to sign up _again_ on this second website and remember another set of usernames and passwords. Giving users two entirely different sets of credentials is not ideal, so that's where Single Sign-On comes in.
+
+Single Sign-On, commonly known as SSO, is an authentication mechanism that enables users to use a single set of credentials to log in to multiple different applications or websites. In a typical SSO setup, you have a service provider—the application the user is trying to log in to—and an identity provider like [FusionAuth](https://fusionauth.io), which handles the authentication. This means your users need only one set of credentials to log into all of your applications.
+
+## Using FusionAuth for SSO
+
+To use SSO, you need to choose an Identity provider which will handle the authentication for you. [FusionAuth](https://fusionauth.io/) is one such identity provider. FusionAuth provides libraries for most major programming languages and frameworks, including PHP, Ruby, Python, Java, Go, and .NET.
+
+FusionAuth is fully customizable and is available as a stand-alone package for Linux, Mac, and Windows, so you can self-host FusionAuth on any server. You can even get started for free, so there's no need to enter your credit card number to follow along with this tutorial.
+
+If you are building a website with WordPress and you need to incorporate user management, you can leverage SSO. It provides a more flexible solution for users and saves them from managing multiple credentials. In this tutorial, you'll learn how to set up Single Sign-On in WordPress using FusionAuth with the help of an [OpenID Connect WordPress plugin](https://wordpress.org/plugins/miniorange-login-with-eve-online-google-facebook/).
```

Comment:
this lands as a little confusing if you don't know what user management is. Can you tie it back to the earlier example of the photo app?